### PR TITLE
fix(TBD-11487): Incorrect jars are requested while adding new CDH 6.1.1 metadata

### PR DESCRIPTION
**Pull request by amedvedenko**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 1]
```diff
@@ -996,15 +996,15 @@
...
         "tagName" : "library"
       }, {
         "childNodes" : [ ],
-        "id" : "DYNAMIC_Cloudera_CDH6_1_1_httpcore_4_4_10_jar",
+        "id" : "DYNAMIC_Cloudera_CDH6_1_1_httpclient_4_5_3",
         "tagName" : "library"
       }, {
         "childNodes" : [ ],
-        "id" : "DYNAMIC_Cloudera_CDH6_1_1_htrace_core4_4_2_0_incubating_jar",
+        "id" : "DYNAMIC_Cloudera_CDH6_1_1_httpcore_4_4_6",
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:1007]: why does 4_4_10 becomes 4_4_10 ?

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 2]
```diff
@@ -996,15 +996,15 @@
         "tagName" : "library"
       }, {
         "childNodes" : [ ],
-        "id" : "DYNAMIC_Cloudera_CDH6_1_1_httpclient_4_5_6_jar",
+        "id" : "DYNAMIC_Cloudera_CDH6_1_1_htrace_core4_4_2_0_incubating_jar",
         "tagName" : "library"
       }, {
         "childNodes" : [ ],
-        "id" : "DYNAMIC_Cloudera_CDH6_1_1_httpcore_4_4_10_jar",
+        "id" : "DYNAMIC_Cloudera_CDH6_1_1_httpclient_4_5_3",
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:1003]: 4_5_6 becomes 4_5_3

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json (hunk 1 above)
Comment (amedvedenko) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:1007]: it's automatically re-generated, I didn't put any manual changes

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json (hunk 2 above)
Comment (amedvedenko) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:1003]: it's automatically re-generated, I didn't put any manual changes

**Review by firefryy — APPROVED** [review]
lgtm

**Review by CarlosTalend — APPROVED** [review]

**Pull request merged**